NOTICE.txt

```diff
@@ -41,6 +41,14 @@ This product includes software from the Ibis project (Apache 2.0)
  * Copyright (c) 2015 Cloudera, Inc.
  * https://github.com/cloudera/ibis
 
+This product includes software from https://github.com/HowardHinnant/date
+ * Copyright (c) 2015, 2016, 2017 Howard Hinnant
+ * Copyright (c) 2016 Adrian Colomitchi
+ * Copyright (c) 2017 Florian Dang
+ * Copyright (c) 2017 Paul Thompson
+ * Copyright (c) 2018 Tomasz Kamiński
+ * Licensed under the MIT License;
```

Comment:
Can you put the license for this in LICENSE.txt instead? There was some early confusion with NOTICE.txt vs. LICENSE.txt but my understanding is that vendored code licenses should be put in the latter

Reply:
for <cstring> and <string.h> agreed, but there seems to be a conflict between the xcode cstring and string.h for some versions so i reverted back to the original header..